<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DLI10-IT-DHRM" name="Default test" tests="2" failures="1" timestamp="6 Nov 2019 08:10:42 GMT" time="25.173" errors="0">
  <testcase name="hybridTestAutomation" time="9.346" classname="com.safexp.MDM.automation.testcases.TestngDemo">
    <failure type="java.lang.AssertionError" message="testcase failedjava.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.AssertionError: testcase failedjava.lang.reflect.InvocationTargetException
	at org.testng.Assert.fail(Assert.java:94)
	at com.safexp.MDM.automation.managerClasses.HybridFrameworkDriver.hybridTestAutomation(HybridFrameworkDriver.java:97)
	at com.safexp.MDM.automation.testcases.TestngDemo.hybridTestAutomation(TestngDemo.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- hybridTestAutomation -->
  <testcase name="hybridTestAutomation" time="13.059" classname="com.safexp.MDM.automation.testcases.TestngDemo"/>
</testsuite> <!-- Default test -->
